mpisppy/spbase.py

```diff
@@ -591,7 +591,7 @@ def report_var_values_at_rank0(self, header="", print_zero_prob_values=False):
                 print("")
 
     def write_first_stage_solution(self, file_name,
-            first_stage_solution_writer=sputils.first_stage_solution_writer):
+            first_stage_solution_writer=sputils.first_stage_nonant_writer):
```

Comment:
Similar here, we should rename the kwarg `first_stage_solution_writer` to `first_stage_nonant_writer`, and maybe rename this method `write_nonant_solution`.